source/MaterialXTest/ShaderValid.cpp

```diff
@@ -1092,7 +1139,8 @@ TEST_CASE("MaterialX documents", "[shadervalid]")
         std::vector<std::string> whiteList =
         {
             "arrayappend", "backfacing", "screen", "curveadjust", "dot_surfaceshader", "mix_surfaceshader"
-            "displacementShader", "displacementshader", "volumeshader", "IM_dot_filename", "ambientocclusion", "dot_lightshader"
+            "displacementShader", "displacementshader", "volumeshader", "IM_dot_filename", "ambientocclusion", "dot_lightshader",
+            "geomattrvalue_integer", "geomattrvalue_boolean", "geomattrvalue_string"
```

Comment:
the geomattrvalue impls are never added to the OSL impl cache even though they are being tested. I think this is due to optimization but not sure.